app/controllers/admin/settings/mascots_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Admin
+  module Settings
+    class MascotsController < Admin::ApplicationController
+      def create
+        errors = upsert_config(settings_params)
```

Comment:
Again, the updates for this specific model seemed simple enough that they don't require a service object. Also, once I'm done extracting all model I'll be looking into further refactoring that can be applied across the board, so this may not be the final form.